pkg/apis/crunchydata.com/v1/doc.go

```diff
@@ -3,7 +3,6 @@ Crunchy PostgreSQL Operator API
 
 The Crunchy PostgreSQL Operator API defines HTTP(S) interactions with the Crunchy PostgreSQL Operator.
 
-
```

Comment:
:question: Same as on the 4.6.x update, updates look good, question about the whitespace changes.